.prow.yaml

```diff
@@ -24,7 +24,7 @@ presubmits:
       preset-goproxy: "true"
     spec:
       containers:
-        - image: golang:1.17.5
+        - image: golang:1.18.8
```

Comment:
We are actually building KubeOne 1.4 with Go 1.18 for quite a while (I'm not sure when we introduced this change exactly). That being said, this is not upgrading the Go version from 1.17 to 1.18, that already happened before.